src/modules/shared/utils/dates.ts

```diff
@@ -27,7 +27,7 @@ export const asDate = (input: Date | string | undefined | null): Date | undefine
 };
 
 export const localisedOptions = {
-	locale: getLocaleFromi18nLanguage('nl'),
+	locale: getLocaleFromI18nLanguage('nl'),
```

Comment:
Wat is de reden dat je hier niet Locale.NL hebt gebruikt?

Reply:
goed punt
ik denk doordat ik vroeger LanguageCode has die 'NL' was, maar ik heb dat dan later omgezet naar Locale 'nl'
ik pas het aan 👍